readme.org

```diff
@@ -534,10 +534,12 @@ more.
 
    Third-party packages are provided by several parties:
 
-   - [[https://github.com/emacs-evil/evil-magit][evil-magit]]
-   - Lispy: [[https://github.com/noctuid/lispyville][lispyville]] or [[https://github.com/sp3ctum/evil-lispy][evil-lispy]]
-   - Org-mode: https://github.com/GuiltyDolphin/org-evil or https://github.com/Somelauw/evil-org-mode
-   - [[https://github.com/atheriel/evil-ledger][evil-ledger]]
+   | Major mode | evil bindings            |
```

Comment:
Maybe we should capitalize evil bindings.

Reply:
It definitely looks better. The page for `evil`/`Evil` isn't entirely clear about what the proper capitalization is.